src/main/java/com/maxio/advancedbilling/models/CouponCurrency.java

```diff
@@ -31,28 +33,53 @@ public CouponCurrency() {
      * Initialization constructor.
      * @param  id  Integer value for id.
      * @param  currency  String value for currency.
-     * @param  price  Integer value for price.
+     * @param  price  Double value for price.
      * @param  couponId  Integer value for couponId.
      */
     public CouponCurrency(
             Integer id,
             String currency,
-            Integer price,
+            Double price,
```

Comment:
Double might be problematic for Currencies. Should be String or BigDecimal.

Reply:
I think sdk doesn't support BigDecimal. We sometimes use oneof: String, Double
I wanted to avoid oneOf matcher for the response.
Should I change it to String?